a10_octavia/controller/worker/flows/a10_load_balancer_flows.py

```diff
@@ -327,8 +327,7 @@ def get_create_rack_vthunder_load_balancer_flow(self, vthunder_conf, topology, l
                 post_amp_prefix, topology, mark_active=(not listeners)))
 
         lb_create_flow.add(virtual_server_tasks.CreateVirtualServerTask(
-            requires=(constants.LOADBALANCER_ID, constants.LOADBALANCER, a10constants.VTHUNDER),
```

Comment:
Why was this change introduced in this PR? If it's a bug blocking this PR then that's fine, but otherwise it should have it's own PR.

Reply:
While testing Rack flows, found this issue while creating LB itself, hence fixed it to move further for fixing the listeners flow of Rack.
Let me know if I should handle this in a separate PR.

Reply:
Okay that's fine to include. Please make sure the commit message surrounding this change explain it though.

Reply:
ok